onebusaway-android/src/main/java/org/onebusaway/android/util/RegionUtils.java

```diff
@@ -277,6 +279,51 @@ public static boolean isRegionUsable(ObaRegion region) {
     }
 
     /**
+     *
+     * @return OTP server URL with trimmed trailing slash, or null if none defined for region.
+     */
+    public static String getOtpBaseUrl() {
+        ObaRegion region = Application.get().getCurrentRegion();
+        String otpBaseUrl = region.getOtpBaseUrl();
+
+        if (otpBaseUrl == null)
+            return null;
+
+        // Trim trailing slash if it exists.
+        return otpBaseUrl.replaceFirst("/$", "");
+    }
+
+    /**
+     *
+     * @return returns true if preferred units are metric, false if Imperial
+     */
+    public static boolean getUnitsAreMetric(Context context) {
```

Comment:
Could you please move this to `PreferenceUtils`?  It really doesn't have anything to do with the region, as it's a user preference.  It should have a name like `getUnitsAreMetricFromPreferences()`.